include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -283,7 +283,11 @@ SECTIONS
 	}
 #endif
 
-#include <app_data_alignment.ld>
+#include <linker_section_alignment.ld>
+
+		. = ALIGN(4);
```

Comment:
I think this can be removed because it will be handled by line 290.

Reply:
Sure? What if _region_min_align is 0?

Reply:
Right, @SebastianBoe , that's why I kept the (4) here. 

However, it is rather unlikely that _region_min_align will be 0 in future MPU architectures.